lib/features/activity/ui/add_activity/add_activity_page.dart

```diff
@@ -135,30 +135,31 @@ class AddActivityPage extends ConsumerWidget {
                     height: 20,
                   ),
                   TextButton(
-                      child: const Text('OK'),
-                      onPressed: () async {
-                        final currentState = formGlobalKey.currentState;
-                        if (currentState == null) {
-                          return;
-                        }
-                        if (currentState.validate()) {
-                          ref
-                              .watch(activitiesListProvider(trip.id).notifier)
-                              .add(
-                                name: activityNameController.text,
-                                activityDate: activityDateController.text,
-                                activityTime: activityTime,
-                                category: activityCategory,
-                                trip: trip,
-                              );
+                    child: const Text('OK'),
+                    onPressed: () async {
+                      final currentState = formGlobalKey.currentState;
+                      if (currentState == null) {
+                        return;
+                      }
+                      if (currentState.validate()) {
+                        ref
+                            .watch(activitiesListProvider(trip.id).notifier)
+                            .add(
+                              name: activityNameController.text,
+                              activityDate: activityDateController.text,
+                              activityTime: activityTime,
+                              category: activityCategory,
+                              trip: trip,
+                            )
+                            .ignore();
```

Comment:
I added these ignores to resolve the lint (unawaited_future), but it's unclear if that's correct. We should probably be monitoring the error here and showing an indication if it fails instead of immediately going to the new page and ignoring the future's result.

Reply:
There's a few spots like this. You can search `.ignore` to see them all